src/calculator.js

```diff
@@ -0,0 +1,84 @@
+import { PropTypes } from 'prop-types';
+import React from 'react';
+
+const Button = (props) => {
+  const { label } = props;
+  return (
+    <button type="button">{label}</button>
+  );
+};
+
+const Screen = () => (
+  <div className="screen-row">
+    <input type="text" value={0} readOnly />
```

Comment:
- [ ] Please make sure that your app has a component `App` that will be the starting point(main component) of that application and call the `calculator` component in the `App` component.

![image](https://user-images.githubusercontent.com/83803414/205931040-a1b16553-9d4b-4b87-9e53-1de3038774c0.png)